src/operator/softmax_activation-inl.h

```diff
@@ -29,9 +29,9 @@ enum SoftmaxActivationOpType {kInstance, kChannel};
 
 struct SoftmaxActivationParam : public dmlc::Parameter<SoftmaxActivationParam> {
   // use int for enumeration
-  int type;
+  int act_type;
   DMLC_DECLARE_PARAMETER(SoftmaxActivationParam) {
-    DMLC_DECLARE_FIELD(type)
+    DMLC_DECLARE_FIELD(act_type)
```

Comment:
act_type sounds weird since it's all softmax. Use a different word?
Also this might break a lot of things